### PR TITLE
Hotfix/0.26.0.1

### LINKED ISSUE #3648: Duplicate the new action menu at the bottom of request pages
After reading a correspondence thread users have to scroll back up to the top to take actions such as replying to it. This suggestion is to duplicate the menu of options at the bottom of the correspondence thread as well as offering it at the top. 

See user feedback at:

https://github.com/mysociety/alaveteli/pull/3451#issuecomment-267313500

### PR DESCRIPTION
**Pull request by garethrees**
Fixes #3648 

screen shot 2017-01-11 at 17 59 36

Requires https://github.com/mysociety/whatdotheyknow-theme/tree/0-26-0-1-updates for WDTK theme

### CODE REVIEW

File: app/views/request/show.html.erb [hunk 1]
```diff
@@ -165,6 +165,17 @@
     </div>
   <% end %>
 
+  <% unless @render_to_file %>
```
Comment (lizconlan) [inline app/views/request/show.html.erb]: Unfortunately, the dropdown to the bottom left of the page doesn't quite work for me (screenshots are themeless to make sure my theme changes weren't causing this).

### Safari (Mac)
screen shot 2017-01-11 at 18 08 06

(the dark grey shadow to the left is the window behind the browser)

### Chrome (Mac)

screen shot 2017-01-11 at 18 16 20

(screen grab includes space to the left and underneath the browser window)

Comment (lizconlan) [inline app/views/request/show.html.erb]: I also didn't find styling the `Follow | 2 followers` bit straightforward in the Italy theme but I only tried it very lazily and that "problem" might be me being tired (or the theme being inconsistent, although as I've been working on it today, functionally that's the same thing)

Comment (lizconlan) [inline app/views/request/show.html.erb]: Please ignore the oversized "Help" link next to "Report this request", that's my browser accessibility setting stepping in and making a mess!

**Comment by lizconlan** [comment]
(out of turn, sorry)

The fix at 64c854f is better but still a little odd as the bottom of the panel ends up flush to the bottom of the window. Probably not important but thought I'd mention it before it slipped my mind.

**Comment by garethrees** [comment]
> bottom of the panel ends up flush to the bottom of the window

Added margin:

screen shot 2017-01-12 at 14 20 01

**Pull request merged**